source/_integrations/airzone.markdown

```diff
@@ -18,13 +18,13 @@ ha_codeowners:
 ha_integration_type: integration
 ---
 
-This integration allows getting values from the local API of Airzone HVAC zoning systems.
+This integration interacts with the Local API of [Airzone HVAC zoning systems](https://www.airzone.es/en/).\
+**Please note that not all Airzone devices have a Local API.**\
+Some devices can only be controlled via the Airzone Cloud API and the oldest ones can only be controlled via Modbus protocol.
```

Comment:
Cool? Now what? As in, what should I do?

Reply:
Removed in ac00452aae956e7b5c512e5752da84675ab84cea